nextjs-app/app/offline/components/offline-projects.tsx

```diff
@@ -90,21 +104,27 @@ function OfflineProject({ i, project, openIndex, setOpenIndex }: {
 
 function ProjectRow({images, text}: ProjectRowProps) {
   return (
-    <div>
-      { images?.length && (
-        <ul>
+    <div className="space-y-8">
+      {images?.length && (
+        <ul className="space-y-4">
           {images.map((image) => {
             return (
-              <li key={image._key} >
-                <Image image={image} />
+              <li key={image._key} className="relative w-full h-[50vh]">
+                <Image 
+                  image={image} 
+                  className="w-full h-full"
+                  objectFit="cover"
+                />
               </li>
             )
           })}
         </ul>
       )}
-      {text?.length && <PortableText
-        value={text as PortableTextBlock[]}
-      />}
+      {text?.length && (
+        <div className="prose max-w-none">
```

Comment:
i never used the `prose` classes before, usually i just use the text sizes set up in the tailwind config and set max-width separately if i want it, but if it does what you want, ok!